experimental/FTheoryTools/src/LiteratureModels/constructors.jl

```diff
@@ -124,6 +218,73 @@ Hypersurface model over a concrete base
 julia> hypersurface_equation_parametrization(h2)
 b*w*v^2 - c0*u^4 - c1*u^3*v - c2*u^2*v^2 - c3*u*v^3 + w^2
 ```
+A yet more special instance of literature model are the F-theory QSMs. Those consist of 708 families of geometries,
+each of which is obtained from triangulations of polytopes in the 3-dimensional Kreuzer-Skarke list. In particular,
+for each of these 708 families, a lot of information is known. Still, those geometries are also somewhat involved.
+Let us demonstrate this on the F-theory QSM based on the 4th polytope in the 3-dimensional Kreuzer-Skarke list.
+We can create and study this model as follows (TODO: currently under development):
+```jldoctest
```

Comment:
```suggestion
```jldoctest; setup = :(artifact"QSMDB")
```
This change loads the artifact before running the doctest. Thus there will no longer be verbose printing of the download during doctest execution

Reply:
Thanks

Reply:
The doctests are still failing, but now with a different error: `LoadError: UndefVarError: `@artifact_str` not defined`.

This seems related to the above suggestion. Any suggestions @lgoettgens ?

Reply:
I can try something tomorrow

Reply:
Let's see if my new fix in [`8923fb7` (#3363)](https://github.com/oscar-system/Oscar.jl/pull/3363/commits/8923fb73b709d6e401a629955189daa44e8e45f5) worked

Reply:
Thank you!